R/wrappers.R

```diff
@@ -182,22 +207,13 @@ t_stat <- function(x, formula,
       broom::glance()
   }
   
-  if (conf_int) {
-    results <- prelim %>%
-      dplyr::select(
-        statistic, t_df = parameter, p_value = p.value, alternative,
-        lower_ci = conf.low, upper_ci = conf.high
-      )
-  } else {
-    results <- prelim %>%
-      dplyr::select(
-        statistic, t_df = parameter, p_value = p.value, alternative
-      )
-  }
-  
-  results %>%
+  # removed unnecessary if(conf_int) clause; only the statistic itself 
```

Comment:
@simonpcouch Could you explain this further? Are we not returning the confidence interval here if the user wants it?

Reply:
Sure thing! It's a bit hard to tell from the diff, but the end of the current [function definition on `master`](https://github.com/tidymodels/infer/blob/fe764792095c61a62fb1fa2fdad550ed4b45a8d6/R/wrappers.R) for `t_stat` looks like this: 

```  
if (conf_int) {
    results <- prelim %>%
      dplyr::select(
        statistic, t_df = parameter, p_value = p.value, alternative,
        lower_ci = conf.low, upper_ci = conf.high
      )
  } else {
    results <- prelim %>%
      dplyr::select(
        statistic, t_df = parameter, p_value = p.value, alternative
      )
  }
  
  results %>%
    dplyr::select(statistic) %>%
    pull()
```

(Lines 185-200, at the end of the `t_stat` definition.) The `prelim` object is created differently based on `conf_int`, but either way, only the `statistic` column is `pull`ed and returned. `t_test` returns essentially the same  `prelim` object without that last `select() %>% pull()`, so it's output _does_ depend on `conf_int`.

So, yes, the confidence interval argument has been ignored in `t_stat`. I assumed this was part of the `t_test` vs. `t_stat` distinction when I first noticed it and removed that code, but I'm glad to adjust the output to depend on `conf_int`! :-)

Reply:
Ah, yes. Definitely just a copy-and-paste error. Thanks!